t/120-templates.t

```diff
@@ -19,7 +19,7 @@ The seven suspects are:
 =end pod
 
 stderr-like { $r = pod2html $=pod[0], :templates<templates> },
-            /'does not contain required templates'/,
+            /''/,
```

Comment:
Shouldn't this include some kind of error message? Any string will always contain the empty string...